LICENSE.mit

```diff
@@ -0,0 +1,7 @@
+Copyright 2021 Brian Douglas
```

Comment:
This should probably be something like the following, because @bdougie didn't make the project, @defunkt [did](https://github.com/github/gitignore/commit/b7cc33a99b02fada900d0e4ba6b7bd38a142f064), and he works for GitHub (helped create it, made the first commit on it, etc.).
```suggestion
Copyright 2010-2021 GitHub, Inc. and contributors
```